components/Reviews/components/AllReviewsLink/index.jsx

```diff
@@ -7,30 +7,38 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { bin2hex } from '@shopgate/pwa-common/helpers/data';
 import I18n from '@shopgate/pwa-common/components/I18n';
 import Link from '@shopgate/pwa-common/components/Router/components/Link';
 import { REVIEW_PREVIEW_LIMIT } from 'Pages/Product/constants';
 import connect from './connector';
-import styles from './style';
+import {
+  containerStyle,
+  linkStyle,
+} from './style';
 
 /**
  * Link to all reviews.
+ * @param {string} productId ProductId.
  * @param {number} totalReviewCount The number of reviews given.
  * @returns {JSX|null}
  */
-const AllReviewsLink = ({ totalReviewCount }) => {
-  if (totalReviewCount < REVIEW_PREVIEW_LIMIT) {
+const AllReviewsLink = ({ productId, totalReviewCount }) => {
+  if (totalReviewCount <= REVIEW_PREVIEW_LIMIT) {
     return null;
   }
-
+  const publicProductId = bin2hex(productId);
```

Comment:
Line breaks missing before and after this line.

Reply:
done